analysis/model/04_01_(a)_cox_fit_model.R

```diff
@@ -205,15 +205,15 @@ coxfit <- function(data_surv, interval_names, covar_names, mdl, subgroup,non_cas
     
     if(non_case_inverse_weight ==1){
       print("Using regular SE's for CI's")
-      results$conf.low=exp(confint(fit_cox_model,level=0.95)[,1]) #use robust standard errors to calculate CI
+      results$conf.low=exp(confint(fit_cox_model,level=0.95)[,1]) #use regular standard errors to calculate CI for non-sampled data
```

Comment:
[Optional] We maybe want an indicator variable that states whether robust SEs were used for the CI calculation but not essential as we should be able to back calculate them.

Reply:
eb8be00f158352b7d593686c4d338beb892552f5

Reply:
Added TRUE/FALSE column to indicated whether data was sampled